package-res/reportviewer/reportviewer-prompt.js

```diff
@@ -344,14 +344,18 @@ define([
       showMessageBox: function( message, dialogTitle, button1Text, button1Callback, button2Text, button2Callback, blocker ) {
         var messageBox = registry.byId('messageBox');
 
-        messageBox.setTitle(dialogTitle);
-        messageBox.setMessage(message);
+        messageBox.setTitle('<div class="pentaho-reportviewer-errordialogtitle">' + dialogTitle + '</div>');
+        messageBox.setMessage('<div class="pentaho-reportviewer-errordialogmessage">' + message + '</div>');
 
```

Comment:
Woha. Now there is some seriously bad coding. But OK, if you want to use raw HTML, then at least do it safely. You must properly encode the contents of "dialogTitle" and "message" or you can easily create HTML and Script injections and other nasty surprises. 

You can use the escapeHTML method of "cdf/dashboard/Utils" for that purpose.